addons/lunch/views/lunch_view.xml

```diff
@@ -31,7 +31,7 @@
 
         <!--View Search to group by employee and input/output (cashmoves)-->
         <record id="view_lunch_employee_payment_filter" model="ir.ui.view">
```

Comment:
You can also rename the view xml ids: view_lunch_cashmove_search, and so on for the other views. It will be cleaner.